purview/utils/general.py

```diff
@@ -33,3 +35,19 @@ def timestamp_to_datetime(timestamp):
                 raise ValueError(error_msg)
 
     raise ValueError(error_msg)
+
+
+def get_all_relationships(item):
+    """
+    Gets all nodes having a relationship with the item node
+    :param item: a node (model) object
+    :return: a list of all nodes having relationship with item node
+    """
+    all_relations = {}
+    for model in all_models:
```

Comment:
As mentioned during our live review, this generates a query to Neo4j per model. So we'll need to refactor this to be more efficient.